### DISCUSSION TITLE
Rack profiles for rack creation

### DISCUSSION THREAD

**Discussion by DenHayes** [netbox-community/netbox#14403] [Ideas]
Has there been any thought given to creating a rack profile that could be used when creating a rack that pre-populates the rack type (2-post, 4-post cabinet, etc), the dimensions, RMU count, and max-depth?

Similar to the Device Type that contains the device profile with size, ports, and power info.

**Comment by abhi1693** [comment]
Something like this https://github.com/netbox-community/netbox/issues/9956?

**Reply by DenHayes** [reply]
Very similar, though I would envision the rack inherited characteristics that could be modified per rack, in-case extenders were added, not moved outright to the profile.

I know I can do this via the API or through an import file and for multiple racks I do.  However, when it's one here and there then it's less hassle to do it via the WebUI.
